knotx-example/knotx-example-monolith/src/test/java/com/cognifide/knotx/monolith/ApplicationTestHelper.java

```diff
@@ -45,16 +45,16 @@ public static void startKnotx() throws Exception {
 
     knotxPort = configuration.getJsonObject("server").getJsonObject("config").getInteger("http.port");
 
-    vertx.deployVerticle(MockRemoteRepositoryVerticle.class.getName(), new DeploymentOptions().setConfig(
-        configuration.getJsonObject("mockRepo")));
+    vertx.deployVerticle(MockRemoteRepositoryVerticle.class.getName(),
```

Comment:
You can make same helper method for verticle deployment as in Standalon - will be less code here.